tests/unit/authors/test_authors_receivers.py

```diff
@@ -61,3 +61,16 @@ def test_name_variations():
             'John Richard Ellis',
             'R Ellis',
             'Richard Ellis'])
+
+
+def test_uuid_generation():
+    json_dict = {
+        "authors": [{
+            "full_name": "John Richard Ellis"
+        }]
```

Comment:
Can you also test with a second author that in case the UUID is there it is left untouched?